### DISCUSSION TITLE
【页面ID】期望模版中新增 notion database 列ID，自动创建Unique ID

### DISCUSSION THREAD

**Discussion by zhynin** [tangly1024/NotionNext#2136] [Ideas]
tangly 你好：

**为什么提出这个新的特性改动**
每次新建博文都需要手动的设置路径：slug，颇为繁琐，期望可以有一个类似WordPress的 https://domain.com/%post_id% 的自动路径。当然也还是可以保留手动设置slug的，例如使用标题拼音。

**描述一下你推荐的解决方案**
新增 notion database 列，使用列ID，自动创建Unique ID，文章和页面都可以Unique ID作为路径，自定义page依旧可以。slug列为空时，默认使用ID作为文章的路径，slug自定义时，slug优先级高于ID，优先显示slug。

**描述一下你考虑过的其它替代解决方案**
利用插件生成UUID，然后插入slug。
新增一列ID，手动拷贝至slug。

**补充说明**
我没有到issues和文档中找到相关的。虽然可以通过自定义数据库表头实现，但是这样就是全局slug替换了，我仅仅想要post文章使用这个功能，page页面依旧是使用自定义的路径。
官方说明：
https://www.notion.so/help/unique-id

**Comment by tangly1024** [comment]
slug字段不是必填，可以留空；将用notion id 作为页面路径。

**Comment by zhynin** [comment]
嗯嗯，也行这样。
只是uuid看起来有点儿凌乱。
谢谢
